Source/Shaders/GlobeVS.glsl

```diff
@@ -175,9 +176,20 @@ void main()
 #endif
 
 #ifdef APPLY_MATERIAL
-    vec3 finalNormal = normalMC;
-    vec3 ellipsoidNormal = normalize(position3DWC.xyz);
-    v_slope = abs(dot(ellipsoidNormal, finalNormal));
-    v_height = height;
+    vec3 northPolePositionMC = vec3(0.0, 0.0, 6356752.3142451793);
```

Comment:
Might be better to use `czm_getWgs84EllipsoidEC().radii.z` rather than hardcoding